doc/api/errors.md

```diff
@@ -612,6 +612,16 @@ A human-readable string describing the reason for the error.
 <a id="nodejs-error-codes"></a>
 ## Node.js error codes
 
+<a id="ERR_ABORT"></a>
+### `ABRT_ERR`
```

Comment:
```suggestion
### `ABRT_ERR`
<!-- YAML
added: v15.0.0
-->
```
?

Reply:
_Technically_ it's not 15.0 and this code existed on DOMException beforehand and users could craft it. The actual non DOMException AbortError is 15.2 - I am not sure what this should say?

Reply:
I think it should say whatever is the first version in which Node.js actually throws errors with this code.

Reply:
What version first had DOMException? 8?

Reply:
I'm a bit confused about this code. Could you point me to where it is defined? I cannot find any occurence of `ABRT_ERR` in the source code.

Reply:
@targos wow, we all missed the typo - it's `ABORT_ERR` and not `ABRT_ERR`.

I'll push a fix, the error is defined in two places - in `internal/errors.j` (as `class AbortError extends Error {`) and in `lib/internal/per_context/domexception.js`.

Reply:
Oh, I didn't even think it could be a typo!

So I checked and v15.0.0 is indeed the first version where we throw errors with that code. I don't think we should mention versions where users could somehow do it themselves.